source/AI.cpp

```diff
@@ -1388,7 +1417,7 @@ void AI::MoveEscort(Ship &ship, Command &command) const
 		if(!dest)
 			// This ship has no route to the parent's destination system, so protect it until it jumps away.
 			KeepStation(ship, command, parent);
-		else if(ShouldRefuel(ship, dest))
+		else if(ShouldRefuel(ship, dest, player))
```

Comment:
Not packaging the player means travel plan consideration probably isn't supported until the NPC Travel pull merges (2990), and we are able to query a ship's `DestinationSystem()` instead of just its `TargetSystem`. For the player's flagship, this could be made to be the front of the TravelPlan.